### ISSUE TITLE
Problem z wystawieniem oferty na produkcyjnym Allegro

### ISSUE THREAD

**Issue by mateuszcom** [allegro/allegro-api#5844]
Mamy problem z wystawieniem jednej oferty na produkcyjnym Allegro. Wszystkie inne oferty wystawiają się poprawnie, w przypadku tej jednej otrzymujemy błąd 500 INTERNAL_SERVER_ERROR. TraceID zapytania: ec9ed7ee8b22a298. Błąd jest przy POST /sale/product-offers. Nie wiemy gdzie szukać problemu, a chcielibyśmy na przyszłość wiedzieć, jak się przed tego typu problemami chronić

**Comment by PrzemyslawLukanowski** [comment]
Problematyczne jest w tym przypadku pole `messageToSellerSettings` - jeśli je usuniesz/wstawisz tam wartość null, problem ustąpi, a wypełnienie tej sekcji przez Kupującego będzie opcjonalne. Jeszcze dziś wprowadzimy poprawkę, która wyeliminuje ten błąd po naszej stronie.

**Comment by PrzemyslawLukanowski** [comment]
Tak, problem z polem `messageToSellerSettings` jest po naszej stronie, Wasz request jest prawidłowy. Jak wspomniałem, jeszcze dziś wprowadzimy poprawkę, która wyeliminuje problem, na który natrafiliście - w pojedynczych przypadkach nadal może występować. Przepraszamy za utrudnienia.

> Z dziwnych rzeczy zauważyliśmy, że nie wiedząc dla czego, wymusiło nam wgrywanie od nowa zdjęć dla danej oferty - jest to o tyle dziwne, że zdjęcia były dzień wcześniej świeżo wgrane do systemu Allegro i nie mogły wygasnąć

Zdjęcia po wgraniu są ważne przez 6 godzin. Jeśli nie zostaną wykorzystane w tym czasie w ofercie, musicie je wgrać ponownie. Chciałbym jednak dodać, że w przypadku zasobów /sale/product-offers nie ma takiej potrzeby, aby wgrywać zdjęcia na nasz serwer - zdjęcia mogą pochodzić z zewnętrznych linków. Więcej informacji na ten temat znajdziecie w [naszym poradniku](https://developer.allegro.pl/tutorials/jak-jednym-requestem-wystawic-oferte-powiazana-z-produktem-D7Kj9gw4xFA#wlasne-zdjecia-i-opis-oferty).
